packages/cli/src/oclif/commands/build.js

```diff
@@ -35,6 +36,11 @@ BuildCommand.flags = buildFlags({
         'Skips installing a fresh copy of npm dependencies on build. Helpful for using `yarn` or local copies of dependencies.',
       hidden: true,
     }),
+    'skip-validation': flags.boolean({
```

Comment:
After writing this change, realized this is the same CLI our dev community uses... (I think). Might be good to suppress the description here... Would love some feedback

Reply:
Same CLI and all the code/discussion (including this one!) is open source! 😁

Reply:
Since you've `hidden: true`, this flag shouldn't show up in docs or help (`zapier help build` shouldn't include it).

Reply:
Oh ya, public repo... hi world 👋 😄 

> Since you've hidden: true

Oh ya, I guess I did think about this when I made the change. The comment just occurred to me as I was taking a fast last pass through my changes. All good then.